src/Cid/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Cid")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Syncromatics")]
+[assembly: AssemblyProduct("Cid")]
+[assembly: AssemblyCopyright("Copyright ©  2017")]
```

Comment:
Is this auto-generated? If not, make this `Copyright (c) 2017 Syncromatics` to match the LICENSE file